langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralModelPermission.java

```diff
@@ -0,0 +1,28 @@
+package dev.langchain4j.model.mistralai;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+class MistralModelPermission {
```

Comment:
```suggestion
public class MistralAiModelPermission {
```

Please make sure that all user-facing classes are `public` and have `MistralAi` (instead of `Mistral`) prefix